src/events/controller/eventController.ts

```diff
@@ -1,43 +1,82 @@
 import {
-  Controller,
   Get,
   Post,
   Body,
   Param,
   Delete,
   Put,
   Inject,
+  Controller,
   UseInterceptors,
   ClassSerializerInterceptor,
 } from "@nestjs/common";
 import { EventsService } from "../service/eventService";
 import { CreateEventDto } from "../dto/eventCreateDto";
 import { UpdateEventDto } from "../dto/eventUpdateDto";
-import { Event } from "../entity/event";
+import { Event, EventPivotDto } from "../entity/event";
 import { eventGetDto } from "../dto/eventGetDto";
-import { SubCriteriaRefDto } from "../dto/subCriteriaRefDto";
+import { idRefDto } from "../dto/idRefDto";
+import { SubCriteriaIdRefDto } from "../dto/subCriteriaIdRefDto";
 import { Period } from "../interface/eventInterface";
+import { logger } from "../../logger";
+import DateCalc from "../utils/eventUtils";
 
 @Controller("events")
 export class EventsController {
   @Inject()
   eventsService: EventsService;
 
+  @UseInterceptors(ClassSerializerInterceptor)
+  @Get("/userRating")
+  async userRating() {
+    // TODO: add user rating here
+    // console.log(DateCalc.getUserRating());
+    // console.log(DateCalc.getUserRating());
+    // const users = await this.eventsService.findAll();
+    // const a = await this.eventsService.userRating();
+    // console.log(a, 2);
+    // return a;
+  }
+
+  @UseInterceptors(ClassSerializerInterceptor)
+  @Get("/ongoing")
+  async Ongoing(): Promise<Event[]> {
+    return await DateCalc.getOngoingEvents();
+  }
```

Comment:
I think we don't need here async-await, Function case is wrong it should start with lowercase, since it's not a class